doc/install_generic.txt

```diff
@@ -0,0 +1,96 @@
+.. include:: css.txt
+
+.. _install_generic:
+
+
+Generic Installation Instructions
+#################################
+
+If you want the latest on the edge installation of Theano, you want `bleeding_edge`_.
+Otherwise go for the `stable`_.
+
+.. note::
```

Comment:
I think this might be beyond the scope and a Theano installation page.
Worst case in a FAQ "Why to I get a network error when I install theano?"